homeassistant/components/bluetooth_le_tracker/device_tracker.py

```diff
@@ -100,6 +118,10 @@ def discover_ble_devices():
             if device.track:
                 _LOGGER.debug("Adding %s to BLE tracker", device.mac)
```

Comment:
Would be nice to keep the same name as above
`address = device.mac[4:]`

Reply:
This log was already there before; I'd prefer not to change this in this PR (it is unrelated). 
Additionally, the mac addresses in "known_devices" are all prefixed with "BLE_", so IMHO it makes sense that the logging shows that as well.

Reply:
```suggestion
                _LOGGER.debug("Adding %s to BLE tracker", device.mac)
                address = device.mac[4:]
```

Reply:
Sorry for the lack of clarity.  When I ran though this, I had to track down what `device.mac[4:]` was doing as its called `address` above.

Reply:
Oh, now I get it ;-)